helmclient.go

```diff
@@ -430,14 +432,26 @@ func (c *Client) GetReleaseHistory(ctx context.Context, releaseName string) (*Re
 	{
 		release := resp.Releases[0]
 
+		var appVersion string
 		var version string
 		if release.Chart != nil && release.Chart.Metadata != nil {
+			appVersion = release.Chart.Metadata.AppVersion
 			version = release.Chart.Metadata.Version
 		}
 
+		var lastDeployed time.Time
+		if release.Info != nil {
+			lastDeployed, err = ptypes.Timestamp(release.Info.LastDeployed)
```

Comment:
This is the protocol buffers magic to convert the timestamp field to a Go time.Time.